services-geojson/src/test/java/com/mapbox/geojson/utils/PolylineUtilsTest.java

```diff
@@ -32,12 +37,67 @@ public void testDecodePath() {
   }
 
   @Test
-  public void testEncodePath() {
+  public void testEncodePath5() {
     List<Point> path = decode(TEST_LINE, Constants.PRECISION_5);
     String encoded = encode(path, Constants.PRECISION_5);
     assertEquals(TEST_LINE, encoded);
   }
 
+  @Test
+  public void testDecodeEncodePath6() {
+    List<Point> path = decode(TEST_LINE6, Constants.PRECISION_6);
+    String encoded = encode(path, Constants.PRECISION_6);
+    assertEquals(TEST_LINE6, encoded);
+  }
+
+  @Test
+  public void testFromPolyline6() {
+
+    List<Point> originalPath = Arrays.asList(
+      Point.fromLngLat(2.2862036, 48.8267868),
+      Point.fromLngLat(2.4, 48.9)
+    );
+    String encoded = encode(originalPath, Constants.PRECISION_6);
+    List<Point> path = LineString.fromPolyline(encoded, Constants.PRECISION_6).coordinates();
+
+    assertEquals(originalPath.size(), path.size());
+    for (int i = 0; i < originalPath.size(); i++) {
+      assertEquals(originalPath.get(i).latitude(), path.get(i).latitude(), 0.000001);
```

Comment:
Let's make this Delta a constant since we use it in so many places for these test.